pytentiostat/routines.py

```diff
@@ -20,10 +20,10 @@ def _load_arduino():
         the COM port the arduino is connected to.
     """
     print("Searching for potentiostat...")
-    ports = list(serial.tools.list_ports.comports())
+    ports = list(serial.tools.list_ports.grep(r'tty|Arduino|COM'))
     n_arduinos = 0
     for p in ports:  # Checking for Arduino Unos connected
-        if "Arduino Uno" in p.description:
+        if re.match(r'tty|Arduino|COM',p.description) is not None:
             com = p.device
             n_arduinos += 1
```

Comment:
I believe this gives false positives of Arduinos connected if this regular expression matches anythin with tty OR Arduino OR COM. Unless I'm reading the expression incorrectly, can we make this more specific so it more precisely identifies if it is an Arduino that is connected?

Reply:
This was done to solve my problem to recognize the board in linux. The identification was performed in the 46 and 47

Reply:
We should move that part into load_arduino (right after this) then cause we iterate number of arduinos here.

Reply:
I have done it to recognize my port